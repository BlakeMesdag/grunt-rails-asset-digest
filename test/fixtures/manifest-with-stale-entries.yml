---
some/assetpipeline/generated-tree.js: some/assetpipeline/generated-tree-536a9e5ddkfjc9v9e9r939494949491.js
another/tree-we-didnt-touch.js: another/entry-we-didnt-touch-536a9e5d711e0593e43360ad330ccc31.js
rootfile.js: rootfile-stale.js
sourcemapping.js.map: sourcemapping-stale.js.map
othersubdirectory/generated-tree.js: othersubdirectory/generated-tree-stale.js
subdirectory/with/alibrary.js: subdirectory/with/alibrary-stale.js
style.css: style-stale.css