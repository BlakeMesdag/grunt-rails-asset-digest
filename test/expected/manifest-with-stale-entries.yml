---
some/assetpipeline/generated-tree.js: some/assetpipeline/generated-tree-536a9e5ddkfjc9v9e9r939494949491.js
another/tree-we-didnt-touch.js: another/entry-we-didnt-touch-536a9e5d711e0593e43360ad330ccc31.js
rootfile.js: rootfile-54267464ea71790d3ec68e243f64b98e.js
sourcemapping.js.map: sourcemapping-742adbb9b78615a3c204b83965bb62f7.js.map
othersubdirectory/generated-tree.js: othersubdirectory/generated-tree-e4ce151e4824a9cbadf1096551b070d8.js
subdirectory/with/alibrary.js: subdirectory/with/alibrary-313b3b4b01cec6e4e82bdeeb258503c5.js
style.css: style-7527fba956549aa7f85756bdce7183cf.css